app/policies/user_policy.rb

```diff
@@ -0,0 +1,30 @@
+class UserPolicy < ApplicationPolicy
```

Comment:
Maybe for this two ACs:  AC:2 User should  be able to receive information only about himself, AC:3 User should be able to receive information only  about his own articles, better to use a `scope`.  

The difference is that: if the action on the resource is prohibited, the user has to receive the 403 response code, and if the resource is not found, 404. In this case, we can adopt a strategy with `scope` because users should not know that other users or articles exist

Reply:
Client error responses: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#client_error_responses